flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
+                          help='Quote to expect in all files (default: \') (deprecated)')
+        parser.add_option('--inline-quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
         parser.config_options.append('quotes')
 
     @classmethod
     def parse_options(cls, options):
-        cls.quotes = cls.QUOTES[options.quotes]
+        cls.inline_quotes = cls.INLINE_QUOTES[options.inline_quotes]
```

Comment:
We need to add a fallback to use `options.quotes` in case `options.inline_quotes` isn't defined